src/app/header/components/ExternalLinks.tsx

```diff
@@ -0,0 +1,12 @@
+import { IconButton } from '@chakra-ui/react';
+import { useState } from 'react';
+import { DiGithubBadge } from 'react-icons/di';
+
+export function ExternalLinks(): JSX.Element | null {
+  const [isOpen, setIsOpen] = useState(false);
+  const open = () => {
+    setIsOpen(!isOpen);
+    window.open('https://github.com/VikeLabs/courseup');
+  };
+  return <IconButton aria-label="GitHub" icon={<DiGithubBadge />} borderRadius="50%" name="github" onClick={open} />;
```

Comment:
Instead of hooking into the state, you can add the props `as="a" href="https://github.com/VikeLabs/courseup"` which will do the same thing. Additionally, if you do it this way, you can add `target="_blank"` so clicking the link opens a new tab.

Reply:
nit: you can use the `isRound` prop instead of doing `borderRadius="50%"` ([chakra docs](https://chakra-ui.com/docs/form/icon-button#props) for reference)

Reply:
my fault for not creating a mock-up to base styling off of, but if we can aim to make it look like the attached screenshot that would be 🔥 
![Screen Shot 2021-06-07 at 5 09 50 PM](https://user-images.githubusercontent.com/53020925/121102737-3a880c00-c7b3-11eb-810c-ce4b56471300.png)

Reply:
The black colour scheme would be better for a11y.
https://contrastchecker.online/000000/82cbee whereas the white variant wouldn't pass.

Reply:
Thanks for the feedback guys! I'll change the code accordingly and request another review.